rpcserverhandlers_test.go

```diff
@@ -1660,6 +1713,42 @@ func TestHandleEstimateFee(t *testing.T) {
 	}})
 }
 
+func TestHandleEstimateSmartFee(t *testing.T) {
+	conservative := types.EstimateSmartFeeConservative
+	economical := types.EstimateSmartFeeEconomical
+	testRPCServerHandler(t, []rpcTest{{
+		name:    "handleEstimateSmartFee: ok with mode",
+		handler: handleEstimateSmartFee,
+		cmd: &types.EstimateSmartFeeCmd{
+			Confirmations: 0,
+			Mode:          &conservative,
+		},
+		result: float64(0),
```

Comment:
It might be worth specifying an estimate fee other than 0, which would test that function is properly converting the returned amount to coins